.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: cpp
+os: linux
+dist: bionic
```

Comment:
`os` and `dist` are set per build job in the matrix configuration below. Apparently those two lines are redundant here and can be removed.

Reply:
True, but Travis keeps throwing warnings if no root level `os` and `dist` are defined.